src/main/java/DateTime.java

```diff
@@ -0,0 +1,32 @@
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class DateTime {
+
+    protected String input;
+    protected Date date;
+
+    public DateTime (String input){
+        this.input = input;
+    }
+
+    public void setDateAndTime() throws DukeException {
```

Comment:
Nice use of the build in Java library. Will definitely consider adding that to my code. However, I am curious as to whether this should be done in the constructor or have a separate function? The errors should be thrown if the input is already wrong in the first place.